### PR TITLE
REM-542

### PR DESCRIPTION
**Pull request by Alinadimmokdad**
iv-settings/
new page

### CODE REVIEW

File: src/pages/iv-settings/form/IvSettings.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  }
+
+  const formik = useFormik({
+    enableReinitialize: true,
+    validateOnChange: true,
+    initialValues,
+    validationSchema: yup.object({
+      iv_minSerialSize: yup
+        .number()
+        .min(1, 'Minimum serial size must be at least 1')
```
Comment (omarZrayka23) [inline src/pages/iv-settings/form/IvSettings.js:63]: can u remove text

File: src/pages/iv-settings/form/IvSettings.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        .max(20, 'Maximum serial size cannot exceed 20')
+    }),
+    onSubmit: values => {
+      postIvSettings(values)
+    }
+  })
+  console.log(formik.values)
```
Comment (omarZrayka23) [inline src/pages/iv-settings/form/IvSettings.js:70]: remove console.log

File: src/pages/iv-settings/form/IvSettings.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+            <ResourceComboBox
+              datasetId={DataSets.ITEM_SEARCH_STYLE}
+              name='itemSearchStyle'
+              label={_labels.itemSearchStyle}
+              valueField='key'
+              displayField='value'
+              values={formik.values}
+              onChange={(event, newValue) => {
+                if (newValue) {
+                  formik.setFieldValue('itemSearchStyle', newValue?.key)
```
Comment (omarZrayka23) [inline src/pages/iv-settings/form/IvSettings.js:106]: remove if condition and use one formik.setFieldValue

**Comment by omarZrayka23** [comment]
Screenshot 2024-09-03 at 12 57 49 PM
big space between field

File: src/pages/iv-settings/form/IvSettings.js (hunk 1 above)
Comment (Alinadimmokdad) [reply src/pages/iv-settings/form/IvSettings.js:63]: its a hidden text

Comment (omarZrayka23) [reply src/pages/iv-settings/form/IvSettings.js:63]: i know but remove if u are not need

**Comment by omarZrayka23** [comment]
decrease hight of window

File: src/pages/iv-settings/form/IvSettings.js [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+            <ResourceComboBox
+              datasetId={DataSets.ITEM_SEARCH_FIELDS}
+              name='itemSearchFields'
+              label={_labels.itemSearchFields}
+              valueField='key'
+              displayField='value'
+              values={formik.values}
+              onChange={(event, newValue) => {
+                if (newValue) {
+                  formik.setFieldValue('itemSearchFields', newValue?.key)
```
[the remark this answers is not in the document]
Comment (Alinadimmokdad) [reply src/pages/iv-settings/form/IvSettings.js]: done

**Pull request merged**